src/Fable.Transforms/Fable2Babel.fs

```diff
@@ -3788,6 +3788,10 @@ module Util =
         (classDecl: Fable.ClassDecl)
         (ent: Fable.Entity)
         =
+        printfn "[classDecl] %A" classDecl
```

Comment:
I assume these were debug statements, can they be removed?

Reply:
Yes sorry!